src/DynamoCoreWpf/UI/Prompts/CrashPrompt.xaml.cs

```diff
@@ -92,7 +103,7 @@ private void Continue_Click(object sender, RoutedEventArgs e)
 
         private void PostOnGithub_Click(object sender, RoutedEventArgs e)
         {
-            DynamoViewModel.ReportABug(this.CrashDetailsContent.Text);
+            DynamoViewModel.ReportABug(markdownPackages);
```

Comment:
Wouldn't this cause that we lose certain details?

Reply:
previously only the stack trace was being sent there but it was not being used

Reply:
I see. Please attach a testing gif. I think these looks good for me